Swiftfin/Components/HTMLFormattedText.swift

```diff
@@ -0,0 +1,62 @@
+//
+// Swiftfin is subject to the terms of the Mozilla Public
+// License, v2.0. If a copy of the MPL was not distributed with this
+// file, you can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) 2024 Jellyfin & Jellyfin Contributors
+//
+
+import SwiftUI
+
+struct HTMLFormattedText: UIViewRepresentable {
```

Comment:
I asked our team and we "officially" support HTML *and* Markdown, so this view would have to accommodate *both*.